jobs/build/advisories/Jenkinsfile

```diff
@@ -0,0 +1,259 @@
+
+        // Expose properties for a parameterized build
+        properties(
+            [
+                buildDiscarder(
+                    logRotator(
+                        artifactDaysToKeepStr: '',
+                        artifactNumToKeepStr: '',
+                        daysToKeepStr: '',
+                        numToKeepStr: '')),
+                [
+                    $class: 'ParametersDefinitionProperty',
+                    parameterDefinitions: [
+                        commonlib.ocpVersionParam('VERSION'),
+                        [
+                            name: 'Z',
+                            description: 'What is the value for Z in version X.Y.Z',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: ''
+                        ],
+                        [
+                            name: 'ASSIGNED_TO',
+                            description: 'To whom the advisories must be assigned',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "openshift-qe-errata@redhat.com"
+                        ],
+                        [
+                            name: 'MANAGER',
+                            description: 'Who is the boss?',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "vlaad@redhat.com"
+                        ],
+                        [
+                            name: 'PACKAGE_OWNER',
+                            description: "Owner's email address",
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "${buildUserEmail}"
+                        ],
+                        [
+                            name: 'IMPETUS',
+                            description: 'For which reason are those advisories being created',
+                            $class: 'hudson.model.ChoiceParameterDefinition',
+                            choices: [
+                                "standard",
+                                "cve",
+                                "ga",
+                                "test",
+                            ].join("\n"),
+                        ],
+                        [
+                            name: 'DATE',
+                            description: 'Format: YYYY-Mon-dd (i.e.: 2050-Jan-01)',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "${dateFormat.format(date)}"
+                        ],
+                        [
+                            name: 'DRY_RUN',
+                            description: 'Take no action, just echo what the job would have done.',
+                            $class: 'hudson.model.BooleanParameterDefinition',
+                            defaultValue: true
```

Comment:
I like this idea which means every time we want to perform the operation we need to uncheck this right?

Reply:
:+1: